src/user.ts

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+  MarqetaError,
+} from './'
+
+import snakecaseKeys from 'snakecase-keys'
+
+export interface UserIdentification {
+  type: string;
+  value: string;
+}
+
+export interface User {
+  token?:string,
+  firstName?: string;
+  lastName?: string;
+  email?:string,
+  password?:string,
+  identifications?: UserIdentification[];
+  birthDate?:string,
+  address1?: string;
+  city?: string;
+  state?: string;
+  country?: string;
+  postalCode?: string;
+  phone?:string;
+  gender?:string,
+  usesParentAccount?: boolean,
+  metadata: any
```

Comment:
You ended almost all but the last three lines with semi-colons `;`... in my `interface` definitions, I've used semi-colons on all lines - even the last, which is something to consider... but here, I'd just say pick a way to handle this, and stick with it for the entire project.

Reply:
oh wow, and `npx tsc` nor `eslint` picked that up!